### ISSUE TITLE
Detection under bet365.com

### ISSUE THREAD

**Issue by rwguerra** [ultrafunkamsterdam/undetected-chromedriver#318]
Hey guys!

I`m facing a problem while accessing http://www.bet365.com with uc v2. I`m able to access the website and if I clear all cookies before login I can even login properly, but somehow, when I try to use any find_element command or get or anything the website detect me somehow and log me off (Im not getting blocked, its logging me off).

Interesting thing is that if I try to access the website from regular chrome I can surf normally (obviously) but if I inspect the page (right mouse click -> inspect) the website log me off too.

It is like they know somehow that we are inspecting the website and acting. I noticed that from every action I do they activate a javascript to check everything, but I couldn't find any code inside js that is related to that (I believe it's there).

The strange thing is that when we inspect website they log you out the same way they are doing through undetected-chromedriver.

From a macOS everything is working good, but from windows, its not. Anybody facing this issue?

**Comment by waffleuk** [comment]
Yes, I’m having similar issues. I seem to be able to log on now, however, if I make any selections, it fails. Looks to me like there’s a recaptcha happening that’s picking up the activity. You’re right about the inspecting elements in standard chrome. That also appears to set of their alarms.

**Comment by espizan99** [comment]
Hi,

I'm facing the same problem.
what it is very strange is that sometimes it works and sometimes no luck.
Anybody has more information about this?

Thanks and best regards,

**Comment by daruizr** [comment]
Hello everyone,

any update? I have the same problem without any solution yet.

**Comment by matthews-g** [comment]
Hello guys!

Step 0: You **MUST** implement your bots logging in. You MUST imitate logging in by interacting directly with the elements, not using cookie sharing or anything like that! You will be IP banned if you do not log in while scraping! 

First, use a small window with mobile user agents. 

Second, I have found out that placing this code before executing any of my methods regarding finding, clicking elements, even navigating, seems to make it work.

`driver.execute_script("window.focus();")`

You **MUST** play with the sleep times in between the different interactions! You should make a function that handles this for you...

```
    def sleep_generator(from_sec=2, to_sec=3):
        return random.randrange(from_sec, to_sec)
```
Also, I highly recommend putting your functions in a try-except loop for at least 20 seconds, stopping only if the method execution was successful.

I can not place any more code here, I had to create my own undetected Chromedriver classes with custom get functions to make it work. 

BET365 is a pain in the ass to make it work, and threads like this are constantly getting monitored by the staff to patch these exploits. I created a Telegram group for sharing ideas "secretly" about this, so we can make the lives of our bots longer:

"https://t.me/joinchat/-Jau2gU8sFBjOTA0"

I think the demand is there for these kinds of groups. If there is already one created, please notify me about it. Thank you!

I will not reply anymore here and will probably delete this comment.

**Comment by skyfyl** [comment]
> any update?

does it work?
